ra2ce/analysis/README.md

```diff
@@ -2,8 +2,8 @@
 
 This module contains all protocols and classes related to performing an analysis on a graph or network.
 Analyses can be identified as:
-- `direct`: calculating the **damages** to the infrastructure (roads) due to a hazard (e.g. flood),
-- `indirect`: calculating the economical **losses** that are a consequence of the damage to the infrastructure.
+- `damages`: calculating the direct damages to the infrastructure (roads) due to a hazard (e.g. flood),
```

Comment:
This is subject of the previous PR.

Reply:
What do you mean? The previous PR was about the `analysis_config_data` subpackage.